docs/tutorial/working_with_equinor_pyscal.rst

```diff
@@ -0,0 +1,88 @@
+Working with the RelPerm class and equinor/pyscal
+======================================
```

Comment:
I believe in ReStructured Text the heading characters (in this case `===`) should be at least as long as the title, and should be followed by a blank line. Same for subheadings below